Libraries/LibJS/Tests/builtins/String/String.js

```diff
@@ -7,3 +7,9 @@ test("typeof", () => {
     expect(typeof String()).toBe("string");
     expect(typeof new String()).toBe("object");
 });
+
+test("string split", () => {
+    expect("foo bar baz".split(" ")).toEqual(["foo", "bar", "baz"]);
+    expect("foo,bar,baz".split(",")).toEqual(["foo", "bar", "baz"]);
+    expect("foo||bar||baz".split("||")).toEqual(["foo", "bar", "baz"]);
+});
```

Comment:
These belong in a separate file `String.prototype.split.js`, look at the other StringPrototype tests for examples.

Reply:
Ok, I'll try to make this changes and work on some error checking. Thanks! :)